sale_order_import/wizard/sale_order_import.py

```diff
@@ -72,18 +77,43 @@ def _parse_file(self, filename, filecontent, detect_doc_type=False):
         logger.debug("Order file mimetype: %s", filetype)
         mimetype = filetype[0]
         supported_types = {
```

Comment:
to ease plugging new import types, this should stay in a separated method :wink: